jupyter_kernel_mgmt/tests/utils.py

```diff
@@ -61,3 +62,7 @@ def execute(code='', kc=None, **kwargs):
         assert execute_input['content']['code'] == code
 
     return msg_id, reply['content']
+
+
+def run_sync(coro_method):
+    return asyncio.get_event_loop().run_until_complete(coro_method)
```

Comment:
Looks like there is a [pytest-asyncio](https://pypi.org/project/pytest-asyncio/#description) plugin which can streamline writing tests for async code.